WeakAuras/AuraEnvironment.lua

```diff
@@ -50,3 +54,138 @@ WeakAuras.helperFunctions = {
   WA_IterateGroupMembers = WA_IterateGroupMembers,
   WA_ClassColorName = WA_ClassColorName,
 }
+
+local LBG = LibStub("LibButtonGlow-1.0")
+WeakAuras.ShowOverlayGlow = LBG.ShowOverlayGlow
+WeakAuras.HideOverlayGlow = LBG.HideOverlayGlow
+
+local function forbidden()
+  prettyPrint(L["A WeakAura just tried to use a forbidden function but has been blocked from doing so. Please check your auras!"])
+end
+
+local blockedFunctions = {
+  -- Lua functions that may allow breaking out of the environment
+  getfenv = true,
+  setfenv = true,
+  loadstring = true,
+  pcall = true,
+  xpcall = true,
+  -- blocked WoW API
+  SendMail = true,
+  SetTradeMoney = true,
+  AddTradeMoney = true,
+  PickupTradeMoney = true,
+  PickupPlayerMoney = true,
+  TradeFrame = true,
+  MailFrame = true,
+  EnumerateFrames = true,
+  RunScript = true,
+  AcceptTrade = true,
+  SetSendMailMoney = true,
+  EditMacro = true,
+  SlashCmdList = true,
+  DevTools_DumpCommand = true,
+  hash_SlashCmdList = true,
+  CreateMacro = true,
+  SetBindingMacro = true,
+  GuildDisband = true,
+  GuildUninvite = true,
+  securecall = true,
+}
+
+local overrideFunctions = {
+  ActionButton_ShowOverlayGlow = WeakAuras.ShowOverlayGlow,
+  ActionButton_HideOverlayGlow = WeakAuras.HideOverlayGlow,
+}
+
+local aura_environments = {};
+WeakAuras.AuraEnvironments = aura_environments
```

Comment:
Same here. I would really prefer if that's never available via WeakAuras. People will try to do funny things with it otherwise.